### PR TITLE
Remove useless MAKE_JOBS var

### PR DESCRIPTION
**Pull request by jschueller**
if necessary it can even be overriden via the env

### CODE REVIEW

**Comment by ocefpaf** [comment]
@mingwandroid are you OK with this solution? Should we merge it?

(Asking b/c I did not participated in the original discussion and I don't know decision made :smile:)

**Comment by jschueller** [comment]
The thing is MAKE_JOBS was only used for Linux, not for osx, so at least it'll be consistent (and simpler). @mingwandroid concern was that you could not easily override this. The thing is you still can, via the env as CPU_COUNT passes trough conda-build.

Anyways this is a minor cleanup, if it doesn't get in I'm not going to insist.

**Comment by mingwandroid** [comment]
IMHO the bug that needs to be fixed is that MAKE_JOBS should be used on macOS, but I too am not going to insist! Merging is fine by me.

**Comment by jschueller** [comment]
MAKE_JOBS  does not pass through conda-build on the contrary to CPU_COUNT, so you may want to open an issue there

**Comment by mingwandroid** [comment]
Not at all. You misunderstand the point here. It is to locally hack build.sh itself during development.

**Comment by jschueller** [comment]
Yes, I don't understand.

**Comment by mingwandroid** [comment]
Merging. There's no point continuing this discussion, you are being willfully obtuse. I explained the purpose behind this in plain English, clearly it is to suit my development methodology. That this isn't the same as yours is neither here nor there.
